### PR TITLE
Fixed role logic and improved styling

### PR DESCRIPTION
**Pull request by lerabo**
-Improved styling;
-Added user role;
-Added navbar logic depending on user role;
-Fixed that when refresh the page, don't log out;
-Got all jobs for frellancer's jobs;
-Added send proposal button only to freelancer's job;

### CODE REVIEW

File: src/components/GoogleAuth/GoogleAuthStyle.ts [hunk 1]
```diff
@@ -27,5 +26,5 @@ export const Typography = styled.div`
 	font-family: Roboto, sans-serif;
 	margin: auto;
 	font-weight: 400;
-	font-size: 16px;
+	font-size: 1.5vw;
```
Comment (lyudad) [inline src/components/GoogleAuth/GoogleAuthStyle.ts]: nonono only px and only from constant file

File: src/components/Layout/Layout.styles.ts [hunk 2]
```diff
@@ -24,4 +24,6 @@ export const Ul = styled.ul`
 export const Li = styled.li`
 	margin-top: 10px;
 	margin-left: 30px;
+	color: ${LIGHTBLUE_COLOR};
+	font-size: 1.5vw;
```
Comment (lyudad) [inline src/components/Layout/Layout.styles.ts]: no

File: src/components/Layout/Layout.tsx [hunk 3]
```diff
@@ -1,37 +1,80 @@
...
+								<>
+									<Li>
+										<NavLink to="post-job">{`${t('Layout.clientTitle')}`}</NavLink>
+									</Li>
+									<Li>
+										<NavLink to="/create-job-post">{`${t('Layout.create')}`}</NavLink>
+									</Li>
+								</>
+							)}
+							{role === 'freelancer' && (
```
Comment (lyudad) [inline src/components/Layout/Layout.tsx]: in condition always use constants

Comment (lerabo) [reply src/components/Layout/Layout.tsx]: used constants

File: src/components/Layout/Layout.styles.ts (hunk 2 above)
Comment (lerabo) [reply src/components/Layout/Layout.styles.ts]: changed

File: src/components/GoogleAuth/GoogleAuthStyle.ts (hunk 1 above)
Comment (lerabo) [reply src/components/GoogleAuth/GoogleAuthStyle.ts]: fixed

**Pull request merged**